jaqpotpy/datasets/graph_pyg_dataset.py

```diff
@@ -22,142 +20,56 @@ def __init__(
         y: Optional[list] = None,
         featurizer: Optional[SmilesGraphFeaturizer] = None,
     ):
-        """The SmilesGraphDataset constructor.
-
-        Args:
-        ----
-            smiles (list): A list of SMILES strings.
-            y (list, optional): A list of target values. Default is None.
-            featurizer (SmilesGraphFeaturizer, optional): A featurizer object for to create graph representations from SMILES strings.
-
-        Example:
-        -------
-        ```
-        >>> from jaqpotpy.jaqpotpy_torch.featurizers import SmilesGraphFeaturizer
-        >>> from rdkit import Chem
-        >>>
-        >>> smiles = ['C1=CN=CN1', 'CCCCCCC']
-        >>> y = [0, 1]
-        >>> featurizer = SmilesGraphFeaturizer()
-        >>> featurizer.add_atom_characteristic('symbol', ['C', 'O', 'Na', 'Cl', 'UNK'])
-        >>> featurizer.add_atom_characteristic('is_in_ring')
-        >>> featurizer.add_bond_characteristic('bond_type', [Chem.rdchem.BondType.SINGLE, Chem.rdchem.BondType.DOUBLE, Chem.rdchem.BondType.TRIPLE])
-        >>>
-        >>> dataset = SmilesGraphDataset(smiles, y=y, featurizer=featurizer)
-        >>> dataset[0]
-        Data(x=[5, 6], edge_index=[2, 10], edge_attr=[10, 3], y=0, smiles='C1=CN=CN1')
-        ```
-
-        """
+        """The SmilesGraphDataset constructor."""
         super().__init__()
-
         self.smiles = smiles
         self.y = y
-        # For the moment there is only one Featurizer for Graph
+        # For the moment there is only one Featurizer for Graph.
         if featurizer:
             self.featurizer = featurizer
         else:
             self.featurizer = SmilesGraphFeaturizer()
             # Default node, edge features in case of not specifying dataset
             self.featurizer.set_default_config()
-
         self.precomputed_features = None
 
-    # def config_from_other_dataset(self, dataset):
-    #     """
-    #     Configures the dataset instance from another dataset.
-
-    #     Args:
-    #         dataset (SmilesGraphDataset): Another dataset to copy the configuration.
-
-    #     Returns:
-    #         SmilesGraphDataset: The configured dataset.
-    #     """
-    #     self.featurizer = SmilesGraphFeaturizer().config_from_other_featurizer(dataset.featurizer)
-    #     return self
-
-    def get_atom_feature_labels(self):
-        """Returns the atom feature labels.
-
-        Returns
-        -------
-            list: A list of atom feature labels.
-
-        """
-        return self.featurizer.get_atom_feature_labels()
-
-    def get_bond_feature_labels(self):
-        """Returns the bond feature labels.
-
-        Returns
-        -------
-            list: A list of bond feature labels.
-
-        """
-        return self.featurizer.get_bond_feature_labels()
-
     def precompute_featurization(self):
         """Precomputes the featurization of the dataset before being accessed by __getitem__"""
-        if self.y:
```

Comment:
This should be included in case we want to create a dataset just to take predictions (no labels are nacessary in this case)